catalog/tests/dags/providers/provider_api_scripts/test_wikimedia_commons.py

```diff
@@ -249,6 +249,33 @@ def test_extract_title_gets_cleaned_title(wmc):
     assert actual_title == expected_title
 
 
+def test_get_record_data_returns_none_when_missing_foreign_identifier(wmc):
+    media_data = _get_resource_json("image_data_example.json")
+    media_data["pageid"] = ""
+
+    record_data = wmc.get_record_data(media_data)
+
+    assert record_data is None
```

Comment:
Can't this be included as a parameter in the test below?

Reply:
It's separate because the level of nesting for the parameters is different: `pageid` is the top-level property, and others are nested in the first item of `imageinfo`. I could combine them, but that would complicate the test functions.